.github/scripts/label_small.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+LABEL_NAME=small
+MAX_SIZE=100
+
+pr_diff=$(gh pr diff $PR_NUM -R $GITHUB_REPOSITORY)
+
+insertions=$(printf "$pr_diff" | grep '^+' | wc -l)
```

Comment:
It seems that printf tries to parse the content of pr_diff, which may lead to invalid formats (see #17338).

Perhaps we should avoid manually parsing the content and use gh pr view instead. For example:

```
pr_diff=$(gh pr view $PR_NUM -R $GITHUB_REPOSITORY --json additions,deletions)

insertions=$(echo "$pr_diff" | jq -r '.insertions')
deletions=$(echo "$pr_diff" | jq -r '.deletions')

total_changes=$((insertions + deletions))
```

@TaiJuWu WDYT? could you open a MINOR to fix it?

Reply:
Thanks for your remainder, will update.